lib/logflare_web/live/search_live/logs_search_lv.ex

```diff
@@ -139,10 +142,13 @@ defmodule LogflareWeb.Source.SearchLV do
           |> assign(:search_op_log_events, search_op_log_events)
           |> assign(chart_aggregate_enabled?: search_agg_controls_enabled?(lql_rules))
 
-        kickoff_queries(source.token, socket.assigns)
+        Process.send_after(self(), :kickoff_queries, 500)
```

Comment:
Instead of `send_after`, you can use `if connected?(socket) do` and only start the queries if connected. :)